admin_manual/configuration_files/external_storage_configuration_gui.rst

```diff
@@ -19,6 +19,10 @@ The External storage support application is enabled on your Apps page.
 Storage configuration
 ---------------------
 
+To access the settings for configuring external storage mounts, click on your Profile icon
+in the top right and select settings from the dropdown.  On the left side under Administration
```

Comment:
at the top right and select Settings from the drop-down menu.  On the left side under Administration